src/app/view/perspectives/table/body/rows/cell-group/data-cell/table-data-cell.component.html

```diff
@@ -8,33 +8,45 @@
                       (edit)="onEdit()">
 </table-data-cell-menu>
 
-<div tableEditableCell
-     [affected]="affected$ | async"
-     [selected]="selected"
-     [readonly]="!allowedPermissions?.writeWithView"
-     [value]="(document | extractValue:column) || (linkInstance | extractValue:column) || ''"
-     (valueChange)="onValueChange($event)"
-     (editKeyDown)="onEditKeyDown($event)"
-     (editStart)="onEditStart()"
-     (editEnd)="onEditEnd($event)"
-     (contextmenu)="onContextMenu($event)"
-     (mousedown)="onMouseDown($event)"
-     (keydown)="onKeyDown($event)"
-     [class.uninitialized-column]="!column?.attributeId"
-     [class.uninitialized-record]="!document?.id && !linkInstance?.id"
-     [style.width]="column | columnWidth:canManageConfig | pixel"
-     class="table-border-right table-border-bottom data-input"
-     data-test="table-data-cell-input">
-</div>
+<!--<div tableEditableCell-->
+<!--[affected]="affected$ | async"-->
+<!--[selected]="selected"-->
+<!--[readonly]="!allowedPermissions?.writeWithView"-->
+<!--[value]="(document | extractValue:column) || (linkInstance | extractValue:column) || ''"-->
+<!--(valueChange)="onValueChange($event)"-->
+<!--(editKeyDown)="onEditKeyDown($event)"-->
+<!--(editStart)="onEditStart()"-->
+<!--(editEnd)="onEditEnd($event)"-->
+<!--(contextmenu)="onContextMenu($event)"-->
+<!--(mousedown)="onMouseDown($event)"-->
+<!--(keydown)="onKeyDown($event)"-->
+<!--[class.uninitialized-column]="!column?.attributeId"-->
+<!--[class.uninitialized-record]="!document?.id && !linkInstance?.id"-->
+<!--[style.width]="column | columnWidth:canManageConfig | pixel"-->
+<!--class="table-border-right table-border-bottom data-input"-->
+<!--data-test="table-data-cell-input">-->
+<!--</div>-->
```

Comment:
Can not we remove that?

Reply:
Yes, of course. I have kept it here in order to see what are all the actions I need to implement and I forgot to remove it afterwards.